interfaces/quizoot.ts

```diff
@@ -243,12 +243,14 @@ export declare namespace Quizoot {
         question_id: string;
         /**
          * Id of the next question in the quiz chronology. Is null if it is the last question.
+         * @nullable
          */
-        next_question_id: string | null;
+        next_question_id: string;
```

Comment:
for this one, its fine when you generate the schema: you have the "null" type. But what about using the types in the components ..? There is no way typescript will know it's nullable.

So I would suggest we keep both : `string | null` and add the tag `@nullable` in the JsDoc.

Reply:
You're right, I didn't take that into account